README.md

```diff
@@ -1,5 +1,59 @@
+propose projects during these meetings, and collaboratively create a list of projects that are of mutual interest. The
+projects include platform work, algorithm development, analytics, visualization, and neurophysiology applications and
+research tasks. Projects are not limited to code development, but can include creation of documentation and tutorial
+materials. Through the course of the meetings, each participant selects one or more project teams and develops goals
+for the Hackathon. The first day of the Hackathon itself begins with a 2-hour in-person introduction to all projects
+and participants. The rest of the Hackathon consists of a mix of working sessions and breakout sessions on special
+topics, as decided by the participants during the preparatory meetings.
+
+**Who can attend?** Hackathons are open to everyone and publicly advertised. One-day participation is permitted for
+first-time attendees, but participants are strongly encouraged to stay for the entire event. Email announcements
+are sent to the [Hackathon mailing list](To be done) (for continuity) and posted on
+the [Hackathon forum](https://github.com/orgs/NeurodataWithoutBorders/teams/hck04/discussions).
+
+**Who should attend?** This is harder to determine. Your best bet is to send an email or attend a preparatory
+videoconference (see below).
+
+**What is the history of NWB hackathons?** The NWB:N team consists of neuroscientists and software developers
+who recognize that creation and adoption of a unified data format is an important step toward breaking down the
+barriers to data sharing in neuroscience. Hackathons are a way for us to collaborate and develop NWB:N as well
+as to engage with the NWB:N user community.  For an overview of NeurodataWithoutBorders see http://nwb.org/.
+
+As part of the development of NWB:N 1.x two hackathons were held at Janelia Farm, in Ashburn, Virginia; the first
+hackathon  on November 20 – 22, 2014 and the second one on May 14-16 at, 2015 (http://crcns.org/NWB/). As part of
+the development of NWB:N 2.0 a first hackathon was held at Janelia Farm, in Ashburn, Virginia  on July 31 - August 1, 2017.
+The primary focus of the first three hackathons has been on development of the NWB:N format as well as on development of a software strategy for NWB:N.
+
+To encourage the development, growth and use of NWB:N as a unified data format for cellular-based neurophysiology
+data, the Allen Institute for Brain Science, Lawrence Berkeley National Lab, and Kitware are organizing a
+development-focused hackathon at the Allen Institute for Brain Science (April 3-6, 2018) and a user-engagement
+and training hackathon at Lawrence Berkeley National Laboratory (April 25-27).
+
+**Who to contact?** Hackathons are led by @xyz, ([Lynne A. Becker, Ph.D](https://alleninstitute.org/what-we-do/brain-science/about/team/staff-profiles/lynne-becker/)) who is happy to tell you more about them.
```

Comment:
@oruebel @mgrauer  What do you think of this ? Who should be the main point of contact ?

Reply:
My guess is that either Lydia or Lynne would be the main POC for the hackathon at Allen. For hackathon at LBNL I will be the main POC.